src/jit/codegenlinear.h

```diff
@@ -183,15 +183,18 @@
     void                genJmpMethod(GenTreePtr jmp);
 
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-    void genGetStructTypeSizeOffset(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR& structDesc,
-                                    var_types* type0,
-                                    var_types* type1,
-                                    emitAttr* size0,
-                                    emitAttr* size1,
-                                    unsigned __int8* offset0,
-                                    unsigned __int8* offset1);
+    void getGetStructTypeOffset(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR& structDesc,
```

Comment:
@CarolEidt and @LLITCHEV - in one the code reviews I have mentioned to Lubo in general we are using "gen" prefix for those routines that generate code (or at least that was my understanding).  I didn't mention that is the CC.  

For this specific case we cannot justify getget based on what i mentioned to Lubo either.  Because this routine doesn't generate any code.  Plus this method is meant to return the struct field types (var_types) which can be thought of as compiler functionality rather than codegen functionality.  Going forward, I foresee that we might need this routine in lowering/rationalizer to do IR transformations. For that reason i have suggested to move this to Compiler